pages/cases/index.jsx

```diff
@@ -6,7 +6,7 @@ const SearchCasesPage = ({ query }) => {
   return (
     <div>
       <NextSeo title="Find unlinked case notes" noindex />
-      <Search query={query} type="cases" />
+      <Search query={query} type="Records" />
```

Comment:
Can we have `records` lowercase so we are consistent.
You also need to change the checks `type === "cases"` in the `Search` component.